### PR TITLE
Fix process58 tests

### PR DESCRIPTION
**Pull request by evgenyz**
This fix would allow us to execute the test suite inside a container. Look into individual commit messages for details.

### CODE REVIEW

**Comment by evgenyz** [comment]
@openscap-ci test this, please

**Review by jan-cerny — COMMENTED** [review]
I think it's mostly fine.

File: tests/probes/process58/dev_to_tty.sh [hunk 1]
```diff
@@ -11,13 +11,15 @@ echo "result file: $result"
 stderr=$(mktemp ${name}.err.XXXXXX)
 echo "stderr file: $stderr"
 
+ptty=`ps 1 | grep " 1 " | awk -F ' ' '{print $2;}'`
```
[the remark this answers is not in the document]
Comment (evgenyz) [reply tests/probes/process58/dev_to_tty.sh:14]: Sure.

File: tests/probes/process58/command_line.sh [hunk 2]
```diff
@@ -5,128 +5,24 @@ set -e -o pipefail
...
-	ESCAPED_CMDLINE_REGEX=$(printf "%s" "bash.*stopped_process.sh.*escaped.?.?*\n.\e")
+pcmdl="oval eval --results $result $srcdir/$name.oval.xml"
 
-########################################################################
-### Wait for start of all processes (all processes have done exec())
-########################################################################
-	wait_for_process $PID
-	wait_for_process ${ZOMBIE_PID} "command_line.sh"
-	wait_for_process ${ESCAPED_PID}
+echo "Eval:"
```
[the remark this answers is not in the document]
Comment (evgenyz) [reply tests/probes/process58/command_line.sh:19]: NP.

File: tests/probes/process58/command_line.sh [hunk 3]
```diff
@@ -5,128 +5,24 @@ set -e -o pipefail
...
+specarg="--report $(echo -ne "\e\n\E[1;33m")\\\n\e"
+specarg_escaped="--report . .[1;33m\\\\n\\e"
 
-	# Run process with special characters in parameters
-	"${PROC}" escaped "$(echo -ne "\e\n\E[1;33m") \\\n\e" &
-	ESCAPED_PID=$!
-	[ -n "${ESCAPED_PID}" ]
-	# "/usr/bin/env bash ./stopped_process.sh escaped . .[1;33m \\n\e"
-	ESCAPED_CMDLINE_REGEX=$(printf "%s" "bash.*stopped_process.sh.*escaped.?.?*\n.\e")
+pcmdl="oval eval --results $result $srcdir/$name.oval.xml"
```
[the remark this answers is not in the document]
Comment (evgenyz) [reply tests/probes/process58/command_line.sh:17]: Yes, it is a good idea, but not abstracted/automated enough. But, use of plain `result.xml` in some of our tests is one of the reasons testsuite fails to be executed concurrently. And manually naming them after the script (to keep them unique) is tedious and error prone.

I would rather have it as `$base_name`, but I would not rename it in a single script for sake of coherency.

**Pull request merged**